api/src/pcapi/scripts/migrate_bene_import.py

```diff
@@ -0,0 +1,152 @@
+from pcapi.models import db
+from pcapi.models.beneficiary_import import BeneficiaryImport
+from pcapi.models.beneficiary_import import BeneficiaryImportSources
+from pcapi.models.beneficiary_import_status import ImportStatus
+
+
+TYPE_ASSO = {
+    BeneficiaryImportSources.demarches_simplifiees.value: fraud_models.FraudCheckType.DMS,
+    BeneficiaryImportSources.jouve.value: fraud_models.FraudCheckType.JOUVE,
+    BeneficiaryImportSources.educonnect.value: fraud_models.FraudCheckType.EDUCONNECT,
+    BeneficiaryImportSources.ubble.value: fraud_models.FraudCheckType.UBBLE,
+}
+
+
+logger = logging.getLogger(__name__)
+
+
+def migrate_bene_import():
+    import_ids = [
+        columns[0]
+        for columns in BeneficiaryImport.query.filter(BeneficiaryImport.source != BeneficiaryImportSources.ubble.value)
+        .with_entities(BeneficiaryImport.id)
+        .order_by(BeneficiaryImport.id.asc())
+        .all()
+    ]
+    index = 0
+    batch_size = 1000
+    created = []
+    updated = []
+    already_fraud_check = []
+    jouve_no_third_party_id = []
+    no_bene_id = []
+    while index < len(import_ids):
+        if index % 1000 == 0:
+            logger.info("Migration script: %s/%s", index, len(import_ids))
+            logger.info("Migration script: created %s", len(created))
+            logger.info("Migration script: updated %s", len(updated))
+            logger.info("Migration script: already_fraud_check %s", len(already_fraud_check))
+            logger.info("Migration script: jouve_no_third_party_id %s", len(jouve_no_third_party_id))
+            logger.info("Migration script: no_bene_id %s", len(no_bene_id))
+
+        imports: list[BeneficiaryImport] = BeneficiaryImport.query.filter(
+            BeneficiaryImport.id.in_(import_ids[index : index + batch_size])
+        ).all()
+        for bene_import in imports:
+            if not bene_import.beneficiaryId:
+                no_bene_id.append(bene_import.id)
+                continue
+
+            if not bene_import.thirdPartyId and not bene_import.applicationId:
+                # some jouve cases
+                fraud_checks = fraud_models.BeneficiaryFraudCheck.query.filter_by(
+                    userId=bene_import.beneficiaryId, type=fraud_models.FraudCheckType.JOUVE
+                ).all()
+                if not fraud_checks:
+                    # only 2 cases in db
+                    jouve_no_third_party_id.append(bene_import.id)
+                    continue
+                third_party_id = f"jouve-beneficiary-import_id_{bene_import.id}"
+            else:
+                third_party_id = bene_import.thirdPartyId or str(bene_import.applicationId)
+                fraud_checks = fraud_models.BeneficiaryFraudCheck.query.filter_by(
+                    userId=bene_import.beneficiaryId, thirdPartyId=third_party_id
+                ).all()
+
+            if any([fraud_check.status for fraud_check in fraud_checks]):
```

Comment:
pylint: 
`[R1729(use-a-generator), migrate_bene_import] Use a generator instead 'any(fraud_check.status for fraud_check in fraud_checks)'`